### ISSUE TITLE
[0.8.2][ANDROID STUDIO Hedgehog | 2023.1.1 Patch 1] java.lang.NoSuchFieldError: Companion

### ISSUE THREAD

**Issue by Tlevas** [Codium-ai/codiumai-jetbrains-release#247]
# User Description
_User did not enter any detailed information._

# Stack Trace
```
java.lang.NoSuchFieldError: Companion
	at ai.codium.webview.WebviewWindowContentPanel.getJcefNotSupportedReason(WebviewWindowContentPanel.kt:89)
	at ai.codium.webview.WebviewWindowContentPanel.<init>(WebviewWindowContentPanel.kt:54)
	at ai.codium.webview.chat.ChatWebviewContentPanel.<init>(ChatWebviewContentPanel.kt:9)
	at ai.codium.webview.chat.ChatWebviewToolWindowFactory$initContent$1$1.invoke(ChatWebviewToolWindowFactory.kt:99)
	at ai.codium.webview.chat.ChatWebviewToolWindowFactory$initContent$1$1.invoke(ChatWebviewToolWindowFactory.kt:96)
	at com.intellij.openapi.application.ActionsKt.invokeLater$lambda$5(actions.kt:58)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:829)
	at com.intellij.openapi.application.impl.ApplicationImpl$3.run(ApplicationImpl.java:454)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:79)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:122)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:41)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:789)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:740)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:734)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:759)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:667)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$7(IdeEventQueue.kt:571)
	at com.intellij.openapi.application.impl.ApplicationImpl.withoutImplicitRead(ApplicationImpl.java:1444)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:571)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:68)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:349)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:348)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:348)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:343)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:995)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:995)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$4(IdeEventQueue.kt:343)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:829)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:385)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

```

# Other Information
```
=== About ===
Build version: Android Studio Hedgehog | 2023.1.1 Patch 1 Build: #AI-231.9392.1.2311.11255304 December 27, 2023
Theme: One Dark Vivid Italic
JRE: 17.0.7+0-b2043.56-10550314, JetBrains s.r.o.
JVM: 17.0.7+0-b2043.56-10550314, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Windows 11.0 (amd64)

=== Plugins ===
Custom plugins: [RoboPOJOGenerator (2.4.3), JetBrains Marketplace Licensing Support (231.9392.3), Atom Material Icons (90.0.0), Nyan Progress Bar (1.14), ADB Wi-Fi (1.2.5), One Dark theme (5.9.0), Codiumate - Code, test and review with confidence - by CodiumAI (0.8.2), Rainbow Brackets (2023.3.7)]
Disabled plugins:[Material Theme UI (8.6.4), Monokai Pro Theme (1.10)]
```

**Comment by ArtsiomCh** [comment]
Thanks for reporting!
Will be fixed in next version.

**Comment by ArtsiomCh** [comment]
Fixed in 0.8.3
